pages/windows/subst.md

```diff
@@ -0,0 +1,15 @@
+# subst
+
+> Associates a path with a drive letter.
+
+- List active substitutions:
+
+`subst`
+
+- Add an association:
+
+`subst {{drive_letter_with_colon}} {{path/to/folder}}`
```

Comment:
Better to give a concrete example with "Z:". Something like - `subst {{Z:}} {{C:\User\Python2.7}}`